docs/cloud/how-to-manage-certificate-filters-in-temporal-cloud.md

```diff
@@ -0,0 +1,71 @@
+---
+id: how-to-manage-certificate-filters-in-temporal-cloud
+title: How to manage certificate filters in Temporal Cloud
+sidebar_label: Manage certificate filters
+description: To limit access to specific CA certificates, you can create certificate filters.
+tags:
+  - how-to
+---
+
+To limit access to specific [CA certificates](/cloud/how-to-manage-certificates-in-temporal-cloud), you can create certificate filters.
+Each filter contains values for one or more of the following fields:
+
+- commonName (CN)
+- organization (O)
+- organizationalUnit (OU)
+- subjectAlternativeName (SAN)
+
+Corresponding fields in the client certificate must match every specified value in the filter.
+In the following example, only the CN field of the certificate's subject is checked, and it must be exactly `code.example.com`.
```

Comment:
Maybe a subheader is appropriate: `Examples` so scanners can pick them up more easily